packages/gatsby-theme-docs/src/layouts/internals/page-content-inset.js

```diff
@@ -15,6 +15,9 @@ import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
 const PageContentInset = styled.div`
   max-width: ${(props) =>
     props.maxWidth || designSystem.dimensions.widths.pageContentWide};
+  section {
```

Comment:
if we're sure it does not lead to accidental scrollbars appearing in certain situations this looks like a better approach than trial-and-erroring offsets.  Consider commenting because we're making use of a side effect, the usage is not about the main purpose of the overflow property.